ztp/gitops-subscriptions/argocd/README.md

```diff
@@ -40,19 +40,76 @@ In order to deploy the OpenShift GitOps operator v1.12 you may apply the provide
 ```
 
 **Steps:**
+
 1. Install the [Topology Aware Lifecycle Operator](https://github.com/openshift-kni/cluster-group-upgrades-operator#readme), which will coordinate with any new sites added by ZTP and manage the application of the PGT-generated policies.
 
-2. Patch the ArgoCD instance in the hub cluster using the patch file previously extracted into the out/argocd/deployment/ directory:
+2. Customize the ArgoCD patch ([link](ztp/gitops-subscriptions/argocd/deployment/argocd-openshift-gitops-patch.json)) for your environment:
+   1. Select the multicluster-operators-subscription image to work with your ACM version. 
+   
+   | OCP version           | ACM version | MCE version | MCE RHEL version | MCE image   |
+   | --------------------- | ----------- | ----------- | -----------------| ----------- |
+   | 4.14/4.15/4.16        | 2.8/2.9     | 2.8/2.9     | RHEL8            | registry.redhat.io/rhacm2/multicluster-operators-subscription-rhel8:v2.8, registry.redhat.io/rhacm2/multicluster-operators-subscription-rhel8:v2.9 |
+   | 4.14/4.15/4.16        | 2.10        | 2.10        | RHEL9            | registry.redhat.io/rhacm2/multicluster-operators-subscription-rhel9:v2.10 |
+   
+   The version of the multicluster-operators-subscription image should match the ACM version. For instance, for ACM 2.10, use the `multicluster-operators-subscription-rhel9:v2.10` image.  
+   For ACM 2.9, use the `multicluster-operators-subscription-rhel8:v2.9` image.  
+   Beginning with the 2.10 release, RHEL9 is used as the base image for multicluster-operators-subscription-.. images. In RHEL9 images, a different universal executable must be copied to work with ZTP. It is located at the following path in the image: `/policy-generator/PolicyGenerator-not-fips-compliant`.  
```

Comment:
`In RHEL9 images, a different universal executable must be copied to work with ZTP`

**Context**: This is not a ZTP limitation. We need to copy the universal executable due to the fact that ArgoCD server image in Openshift GitOps Operator uses builder image based in Rhel8.  So, when argocd invokes the kustomize to generate manifests, due to the version dependencies of GNU C library(Glibc) in rhel8 and rhel9, there occurs a mismatch. [Rhel8 ships with glibc-2.28 and Rhel9 contains glibc-2.34](https://access.redhat.com/solutions/38634)
The reason, ZTP is not moving to adopt the rhel9 based image due to the above mentioned issue. 

Please rephrase the dependency towards GitOps operator and not ZTP.

Reply:
fixed